checks.d/vsphere.py

```diff
@@ -700,12 +707,20 @@ def _cache_morlist_process_atomic(self, instance, mor):
             " for MOR {0} (type={1})".format(mor['mor'], mor['mor_type'])
         )
 
+        if mor['mor_type'] in REALTIME_RESOURCES:
+            interval = REAL_TIME_INTERVAL
+        else:
+            interval = None
+
         available_metrics = perfManager.QueryAvailablePerfMetric(
-            mor['mor'], intervalId=REAL_TIME_INTERVAL)
+            mor['mor'], intervalId=interval)
 
-        mor['metrics'] = self._compute_needed_metrics(instance, available_metrics)
-        mor_name = str(mor['mor'])
+        if available_metrics:
```

Comment:
You're right, we probably don't need it, going to try with passing around the empty list.